tests/integration/review_app/test_app_df_graph.py

```diff
@@ -152,5 +152,8 @@ def update_graph(rows, selected_rows):
     dash_duo.wait_for_element("#waitfor")
 
     dash_duo.wait_for_element("#{}".format(IDS["table"]))
+    dash_duo.wait_for_element("#pop svg")
+    dash_duo.wait_for_element("#lifeExp svg")
+    dash_duo.wait_for_element("#gdpPercap svg")
```

Comment:
Slight changes in timing / rendering require the test to be more specific and wait for the graph to be ready